plugins/polio/api/vaccines/vaccine_authorization.py

```diff
@@ -247,6 +253,24 @@ def get_most_recent_authorizations(self, request):
 
                 response.append(vacc_auth)
 
+        if auth_status:
+            response = [entry for entry in response if entry["status"] in auth_status.split(",")]
+
+        if block_country:
+            block_country = block_country.split(",")
+            block_country = Group.objects.filter(pk__in=block_country)
+            org_units = [
+                ou_queryset
+                for ou_queryset in [
+                    ou_group_queryset for ou_group_queryset in [country.org_units.all() for country in block_country]
+                ]
+            ]
+            ou_pk_list = []
+            for ou_q in org_units:
```

Comment:
I maybe wrong, but this can be done on one line of code. thank you

Reply:
indeed it can be done with a list comprehension